rfc-drafts/stream.md

```diff
@@ -93,6 +91,28 @@ The arguments to `poll_next` match that of the [`Future::poll`] method:
 [context]: https://doc.rust-lang.org/std/task/struct.Context.html
 [Waker]: https://doc.rust-lang.org/std/task/struct.Waker.html
 
+### Why does next require Self:Unpin?
```

Comment:
I'm not sure this belongs in the Guide-level explanation, it seems like it'd be a better fit in the section dedicated to Next below.